src/libYARP_OS/include/yarp/os/Value.h

```diff
@@ -276,6 +276,16 @@ class YARP_OS_API yarp::os::Value : public Portable, public Searchable {
         return asString() != alt;
     }
 
+    bool operator==(const int alt) const
```

Comment:
Have you evaluated the impact of removing the == operator (simply counting the number of uses (i.e. errors) by commenting out the operator?